src/test_scenes/sceneProgressBar.js

```diff
@@ -38,15 +38,23 @@ class SceneProgressBar extends React.Component {
   render() {
     const { value1, value2, value3 } = this.state;
     return (
-      <View position={this.props.position}>
+      <LinearLayout 
+        defaultItemAlignment={Alignment.centerCenter}
+        defaultItemPadding={[0.05, 0, 0, 0]}
+        itemPadding={[
+          { index: 0, padding: [0, 0, 0.1, 0] },
+          { index: 4, padding: [0.15, 0, 0, 0] },
+        ]}
+        position={this.props.position}
+      >
         <ProgressBar 
-          alignment={'center-center'}
+          alignment={Alignment.centerCenter}
           position={[0, 0.2, 0]} 
```

Comment:
do you still need the positions for all the progress bars?  You used them when it was a `View` but now `LinearLayout` doesn't need them.

Reply:
right 🙇

Reply:
Fixed (removed values for `position` and `alignment` props).